core/keyserver/keyserver.go

```diff
@@ -190,29 +183,35 @@ func (s *Server) getEntry(ctx context.Context, domainID, userID, appID string, f
 				LeafValue: leaf,
 			},
 		},
-		Smr:            getResp.GetMapRoot(),
-		LogRoot:        logRoot.GetSignedLogRoot(),
-		LogConsistency: logConsistency.GetProof().GetHashes(),
-		LogInclusion:   logInclusion.GetProof().GetHashes(),
+		Smr:          getResp.GetMapRoot(),
+		LogInclusion: logInclusion.GetProof().GetHashes(),
 	}, nil
 }
 
 // ListEntryHistory returns a list of EntryProofs covering a period of time.
 func (s *Server) ListEntryHistory(ctx context.Context, in *pb.ListEntryHistoryRequest) (*pb.ListEntryHistoryResponse, error) {
 	// Lookup log and map info.
-	domain, err := s.domains.Read(ctx, in.DomainId, false)
+	domainID := in.GetDomainId()
```

Comment:
GetDomainId() - should be GetDomainID().
Also looks like you often get the id then do domains.Read() on it, which might be a helper perhaps?

Reply:
`GetDomainId` is a function generated from protobuf. The protobuf people decided to _not_ follow Go's standard for initialisms unfortunately: golang/protobuf#156.

The emerging pattern is that all functions supply `domain_id` and the first thing they do is `domains.Read`. Not sure what the best way to pull out a helper.  It feels more like an interceptor that puts something in `ctx`, but that feels a little heavy handed and makes the code path harder to follow.

Reply:
Ah, well it was just a suggestion.